articles/quickstart.md

```diff
@@ -77,7 +84,12 @@ Applications developed with Microsoft's Quantum Development Kit consist of two p
 
 ## Write a Q# operation
 
-Our goal is to prepare two qubits in a [Bell state](https://en.wikipedia.org/wiki/Bell_state) showing entanglement. We will build this up piece by piece to demonstrate qubit states, gates, and measurement.
+Our goal is to prepare two qubits in a [Bell state](https://en.wikipedia.org/wiki/Bell_state) showing entanglement.  Our goal is to prepare two qubits in a specific quantum state, demonstrating how to operate on qubits with Q# to change their state and demonstrate the effects of superposition and entanglement. We will build this up piece by piece to demonstrate qubit states, operations, and measurement.
+
+**Overview:**  In the first code below, we show you how to work with qubits in Q#.  We’ll introduce two operations, M and X that transform the state of a qubit. 
```

Comment:
```suggestion
**Overview:**  In the first code below, we show you how to work with qubits in Q#.  We’ll introduce two operations, [M](xref:microsoft.quantum.intrinsic.m) and [X](xref:microsoft.quantum.intrinsic.x) that transform the state of a qubit. 
```

Reply:
Normally, I'm a bit fan of hyperlinks to further detail.  But not in this first leaner article.  The link does not lead to a page that will further inform the new learner.  It's not time to introduce them to the formal definitions of operations, imho.